### PR TITLE
Update .extend() announcement blog

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
@palnabarun @sayanchowdhury Lazy consensus to merge before 10 pm IST today for PPG

File: content/blog/2020/trying-something-new.md [hunk 1]
```diff
@@ -2,38 +2,36 @@ Title: Trying something different this year
...
 
-### Adding an additional track to the conference
+## Our first order of business, adding another track!
 
-PyCon India 2020, over the last few years, has been a three-track event. Seeing the quality of the talks, the team decided to add an additional track for the event this year.
+Over the last few years, PyCon India 2020 has been a three-track event. Seeing the quality of the talks this year, we felt inclined to make this year's PyCon India's lineup our biggest ever with an additional track for the event this year! 
 
-So, PyCon India 2020 will be a four-track event. This would mean the event would have even more amazing speakers presenting their exceptional talks at the conference. The talks would range from beginner to advanced and cover a wide range of computer science topics.
+Hence, PyCon India 2020 will now be a four-track event effectively increasing talks selected for the conference by 33%. This would mean more people getting the opportunity to present, and more talks for us to watch right at our home. Right? Right? We didn't stop there, presenting...!
```
Comment (Sc0rpi0n101) [inline content/blog/2020/trying-something-new.md:19]: 5 track? 
Including .extend we have 5. Saying 4 and announcing 5th sounds confusing as @inovizz said.

Comment (vipulgupta2048) [reply content/blog/2020/trying-something-new.md:19]: Read the next paragraph.

Comment (vipulgupta2048) [reply content/blog/2020/trying-something-new.md:19]: We are going for an incremental change in our tracks. Hence we first introduce the new 4th track to warm audiences up of how it was before and then introduce something entirely new.

**Review by sayanchowdhury — APPROVED** [review]

File: content/blog/2020/trying-something-new.md (hunk 1 above)
Comment (sayanchowdhury) [reply content/blog/2020/trying-something-new.md:19]: Yes, I agree with @vipulgupta2048 here. It's incremental. Contextual understanding

**Pull request merged**